src/test/java/org/cactoos/io/WriterAsOutputStreamTest.java

```diff
@@ -125,9 +124,7 @@ public void writesLargeContentToFile() throws IOException {
     public void writesToFileAndRemovesIt() throws Exception {
         final Path temp = this.folder.newFile().toPath();
         final String content = "Hello, товарищ! How are you?";
-        try (final OutputStreamWriter writer = new OutputStreamWriter(
-            new FileOutputStream(temp.toFile()), StandardCharsets.UTF_8
-        )) {
+        try (Writer writer = Files.newBufferedWriter(temp)) {
```

Comment:
@ilyakharlamov please, check if is possible another kind of object instead of use a static method.